common/djangoapps/student/views.py

```diff
@@ -735,6 +736,8 @@ def dashboard(request):
         'course_programs': course_programs,
         'disable_courseware_js': True,
         'xseries_credentials': xseries_credentials,
+        'use_ecommerce_payment_flow': EcommerceService().is_enabled(),
```

Comment:
I noticed that in course_modes.views.py above the 'use_ecommerce_payment_flow' parameter is conditionally available based on sku, versus in this view the parameter always available.  Does it make sense for them to both work the same way?  This also goes for 'ecommerce_payment_page'